### ISSUE TITLE
Audible adjustment confirmation

### ISSUE THREAD

**Issue by d4rken** [d4rken-org/bluemusic#3]
> Great app ! I wanted to build one exactly like that for myself. Guess you beat me to it :-) . 2 minor things.... If possible, in a future release to play a confirmation sound once the app is done adjusting the volume . Also, ajusting the volume manually, after the app will also change the values in the app settings. It would be nice to have an option to leave the pre-sets untouched if volume is manually adjusted after.
